tests/integration/server.test.js

```diff
@@ -16,7 +16,7 @@ describe('Root directory Test', () => {
   });
 
   it('Should return status: 200', (done) => {
-    agent.get('/api/v1').end((err, res) => {
+    agent.get('/').end((_err, res) => {
```

Comment:
why the underscore?

Reply:
@Temmyogunbo Preventing eslint from flagging it as a syntax error